src/main/java/ulcrs/models/session/Session.java

```diff
@@ -2,53 +2,38 @@
 
 import ulcrs.models.schedule.Schedule;
 
-import java.time.LocalDateTime;
+import java.util.Date;
 
 public class Session {
 
     private String name;
     private Schedule existingSchedule;
-    private LocalDateTime generatedTimestamp;
-    private LocalDateTime lastEditedTimestamp;
+    private Date createdTimestamp;
+    private Date lastEditedTimestamp;
 
-    public void save() {
-        // TODO: implement
-    }
-
-    public boolean saveAs(String filename) {
-        // TODO: implement
-        return false;
+    public Session() {
+        lastEditedTimestamp = createdTimestamp = new Date();
```

Comment:
Nitpick: break up into two lines? Not vital and is mostly style, so this is totally up to you

Reply:
Done